Dockerfile

```diff
@@ -11,8 +12,6 @@ RUN DEBIAN_FRONTEND=noninteractive && apt-get update && apt-get dist-upgrade -y
   g++ \
   git \
   python-dev \
-  python-pip \
```

Comment:
These are now installed using pip to get the same versions on all Python versions, and not relaying on Debian packages